taca/storage/storage.py

```diff
@@ -114,3 +218,31 @@ def _send_to_swestore(f, dest, remove=True):
         else:
             LOG.info("Run {} is not {} days old yet. Not archiving".format(run, str(days)))
     os.remove("{}.archiving".format(run.split('.')[0]))
+
+
+def get_closed_projects(projs,pj_con,days,logger=None):
```

Comment:
@guillermo-carrasco : Thats my reason why I passed `logger` as argument :) But do you prefer to initiate logger within the new method instead of passing it as argument ??